.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,32 @@
-**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**
+**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`
 
-***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***
+`{Please ensure that your pull request is for the 'development' branch!}`
 
-- [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
-- [] I have written tests and verified that they fail without my change.
-- [] I have squashed any insignificant commits.
-- [] This change has comments for package types, values, functions, and non-obvious lines of code.
+- [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
+- [] I have made only one major change in my proposed changes.
+- [] I have commented my proposed changes within the code.
+- [] I have tested my proposed changes, and have included unit tests where possible.
 - [] I am willing to help maintain this change if there are issues with it later.
-- [] I give this submission freely and claim no ownership. It is compatible with the EUPL 1.2 license.
-- [] I have Signed Off all commits. (`git commit --signoff`)
+- [] I give this submission freely and claim no ownership.
+- [] It is compatible with the EUPL 1.2 license.
```

Comment:
https://opensource.org/licenses/EUPL-1.1 Link for EUPL (Yes, it actually is 1.2, the link is correct.)